src/main/java/pages/MenuPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+
+public class MenuPage extends BasePage {
+
+    public MenuPage() {
+        super();
+    }
+
+    @AndroidFindBy(xpath = "//*[@content-desc='WebView']")
+    private MobileElement bottomBarWebView;
+
+    @AndroidFindBy(xpath = "//*[@content-desc='Login']")
+    private MobileElement bottomBarLogin;
+
+    @AndroidFindBy(xpath = "//*[@content-desc='Home']")
+    private MobileElement bottomBarHome;
+
+    @AndroidFindBy(xpath = "//*[@content-desc='Forms']")
+    private MobileElement bottomBarForms;
+
+    @AndroidFindBy(xpath = "//*[@content-desc='Swipe']")
+    private MobileElement bottomBarSwipe;
+
+    public void clicWebViewMenu() {
+        bottomBarWebView.click();
+    }
+
+    public void clicLoginMenu() {
+        bottomBarLogin.click();
+    }
+
+    public boolean isDisplayedHomeMenu(){
+        return bottomBarHome.isDisplayed();
+    }
+
+    public boolean isDisplayedWebViewMenu(){
+        return bottomBarWebView.isDisplayed();
+    }
+
+    public boolean isDisplayedLoginMenu(){
+        return bottomBarLogin.isDisplayed();
+    }
+
+    public boolean isDisplayedFormsMenu(){
+        return bottomBarForms.isDisplayed();
+    }
+
+    public boolean isDisplayedSwipeMenu(){
+        return bottomBarSwipe.isDisplayed();
```

Comment:
.isDisplayed() is a function from the framework, you don`t need to create a function to call another function for each element. 2 options here. Justo call the element from the step y there you call to is displayed directly. Or you can create a function to validate that all the elements are displayed when the page is loaded(just to validate each time you enter the pague that all the elements are shown). 
In wdio i do something like this in each page(and i call this when the page should load in the steps): 

`validate(){
try{
element1.waitForDisplayed();
element2.waitForDisplayed();
.
.
.
}
catch{throw error}
}`

Reply:
I changed the method for this:
<code>public boolean isDisplayedMenu(){
        return bottomBarHome.isDisplayed() && bottomBarWebView.isDisplayed()
                && bottomBarLogin.isDisplayed() && bottomBarForms.isDisplayed()
                && bottomBarSwipe.isDisplayed();
    }</code>